tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py

```diff
@@ -1,9 +1,16 @@
 import logging
 import itertools
+import re
 from typing import Optional
 
 import torch
 from torch.testing._internal import common_utils
+try:
+    import torch_ucc
+except ImportError:
+    HAS_TORCH_UCC = False
+else:
+    HAS_TORCH_UCC = True
```

Comment:
this looks kind of redundant to me and I feel like getting this from `distributed_test_base.py`